ls1intum/Artemis
# `Programming exercises`: Improve configuration of Gradle projects

### Checklist
#### General
#### Server
#### Client
#### Changes affecting Programming Exercises

### Motivation and Context
This PR contains multiple small improvements regarding the Gradle project type:
- Adding project name to `settings.gradle`
- Change project name in `settings.gradle` on importing existing gradle exercise
- Hide "update template" option on importing Gradle exercises
- Adjust readme.md describing the process for instructors to open all three repositories in one project in the local IDE
- Programming Exercise Documentation

### Steps for Testing
Prerequisites:
- 1 Instructor

1. Log in to Artemis
2. Navigate to Course Administration
3. Create a new Programming Exercise of type 'Gradle - Gradle'
4. Clone all three repositories and follow the description in the test-repository-projects' readme.md
5. Check that the classes from the template/solution-project are recognized (compiled tbp) in the test project
6. Go back to Artemis and import this exercise with a different project name
7. Confirm that the project name has changed in the `settings.gradle` for all three projects
